docs/root/operations/admin.rst

```diff
@@ -194,6 +194,16 @@ modify different aspects of the server:
   field, use the mask query parameter documented above. If you want only a subset of fields from the repeated
   resource, use both as documented below.
 
+.. _operations_admin_interface_config_dump_by_name_regex:
+
+.. http:get:: /config_dump?name_regex={}
+
+  Dump only the currently loaded configurations whose names match the specified regex. Can be used with
```

Comment:
doc nit: Can you ref link some example name fields for the user so it's more clear what name we are referring to. Separately, this means that effectively we need a `name` field in all xDS protos to make this consistent, right? (Just trying to make sure I understand.)

Reply:
So currently all xDS protos have a name-like field; most of those name-like fields are actually called `name`, except for endpoints, which have `cluster_name`. Only the protos whose name-like field matches the given regexp will be added to the dump. E.G. putting in `name_regex=prefix.*` will return routes such that `route.name()` matches, and endpoints such that `endpoint.cluster_name()` matches.

Reply:
Yup that makes sense. I would just ref link some fields or just list them in a bullet list to be more clear to the user. Right now I think it might not be clear to the user what this will do.

Reply:
Done, for some reason clang-format is complaining about this file and `tools/code_format/check_format.py fix` isn't making any changes -- do you know what that's about?

Reply:
Hmm, not sure how it would be complaining about the RST file. What is the error? cc @phlax

Reply:
Wound up forcing with --no-verify, but the error was 
```
~/git/envoy$ git push origin FilterConfigDump
Running pre-push check; to skip this step use 'push --no-verify'
  Checking format for docs/root/operations/admin.rst - ERROR: From ./docs/root/operations/admin.rst
ERROR: Unable to find Envoy namespace or NOLINT(namespace-envoy) for file: ./docs/root/operations/admin.rst
ERROR: clang-format check failed for file: ./docs/root/operations/admin.rst
ERROR: check format failed. run 'tools/code_format/check_format.py fix'
error: failed to push some refs to 'github.com:paul-r-gall/envoy.git
```

Reply:
Hmm this sounds like some type of check format regression. cc @phlax to help out.